profiles/default/linux/amd64/package.use.force

```diff
@@ -0,0 +1,6 @@
+# Copyright 2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+# Eric Joldasov <bratishkaerik@landless-city.net> (2025-03-08)
+# "x86_64" backend can succesfully bootstrap itself for Linux.
+>=dev-lang/zig-0.13 -llvm
```

Comment:
I think you should be doing this in `profiles/arch/base` and `profiles/arch/amd64`.

Reply:
> I think you should be doing this in `profiles/arch/base` and `profiles/arch/amd64`.

Thank you. I feel uneasy creating new file in `profiles` directory, but guess it's unavoidable in both versions.